ddtrace/llmobs/_integrations/anthropic.py

```diff
@@ -0,0 +1,121 @@
+import json
+from typing import Any
+from typing import Dict
+from typing import Iterable
+from typing import Optional
+
+from ddtrace._trace.span import Span
+from ddtrace.internal.logger import get_logger
+from ddtrace.llmobs._constants import INPUT_MESSAGES
+from ddtrace.llmobs._constants import METADATA
+from ddtrace.llmobs._constants import MODEL_NAME
+from ddtrace.llmobs._constants import OUTPUT_MESSAGES
+from ddtrace.llmobs._constants import SPAN_KIND
+
+from .base import BaseLLMIntegration
+
+
+log = get_logger(__name__)
+
+
+API_KEY = "anthropic.request.api_key"
+MODEL = "anthropic.request.model"
+TOTAL_COST = "anthropic.tokens.total_cost"
+TYPE = "anthropic.request.type"
+
+
+class AnthropicIntegration(BaseLLMIntegration):
+    _integration_name = "anthropic"
+
+    def llmobs_set_tags(
+        self,
+        span: Span,
+        input_messages: Optional[Dict[str, Any]] = None,
+        formatted_response: Optional[Dict[str, Any]] = None,
+        err: bool = False,
```

Comment:
I would advise being consistent with the OpenAIIntegration.llmobs_set_tags(...) function signature, because 1) it'll keep maintainability easier if everything is the same pattern, and 2) keeping both `resp` and `streamed_completions` may be easier to work with since we'll be able know to handle streamed vs non-streamed responses.